user-interface/src/staff-assignment/modal/AssignAttorneyModal.test.tsx

```diff
@@ -179,7 +200,9 @@ describe('Test Assign Attorney Modal Component', () => {
     const modalRef = React.createRef<AssignAttorneyModalRef>();
     renderWithProps(modalRef);
 
+    const callback = vi.fn();
```

Comment:
this looks like it is used everywhere. Define in beforeEach?

Reply:
Yes.